commerce-address-content-web/src/main/java/com/liferay/commerce/address/content/web/internal/display/context/CommerceAddressDisplayContext.java

```diff
@@ -94,6 +94,15 @@ public String getAddCommerceAddressURL() {
 		return portletURL.toString();
 	}
 
+	public CommerceAccount getCommerceAccount() throws PortalException {
+		ThemeDisplay themeDisplay =
+			(ThemeDisplay)_httpServletRequest.getAttribute(
+				WebKeys.THEME_DISPLAY);
+
+		return CommerceAccountLocalServiceUtil.getPersonalCommerceAccount(
```

Comment:
@alecsloan can we swich this to use current account from account helper ?

Reply:
See new commit